internal/operators/ocs/validation_test.go

```diff
@@ -1051,6 +1051,39 @@ var _ = Describe("Ocs Operator use-cases", func() {
 			}),
 			errorExpected: false,
 		},
+		{
+			name:               "ocs enabled, 6 nodes with 3 worker nodes for standard deployment",
+			srcState:           models.ClusterStatusReady,
+			dstState:           models.ClusterStatusReady,
+			machineNetworkCidr: "1.2.3.0/24",
+			apiVip:             "1.2.3.5",
+			ingressVip:         "1.2.3.6",
+			dnsDomain:          "test.com",
+			pullSecretSet:      true,
+			hosts: []models.Host{
+				{ID: &hid1, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid2, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid3, Status: swag.String(models.HostStatusKnown), Inventory: defaultInventory(), Role: models.HostRoleMaster},
+				{ID: &hid4, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+				{ID: &hid5, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+				{ID: &hid6, Status: swag.String(models.HostStatusKnown), Inventory: ocsInventoryWithDisks(10, 32000000000), Role: models.HostRoleWorker},
+			},
+			statusInfoChecker: makeValueChecker(clust.StatusInfoReady),
+			validationsChecker: makeJsonChecker(map[clust.ValidationID]validationCheckResult{
+				clust.IsMachineCidrDefined:                {status: clust.ValidationSuccess, messagePattern: "The Machine Network CIDR is defined"},
```

Comment:
Please rename `clust` to readable name `cluster`.

Reply:
We have cluster defined https://github.com/openshift/assisted-service/blob/0076817559528d2d4c266ddc9dda5e7392ece26b/internal/operators/ocs/validation_test.go#L279 here, can you suggest a different name?

Reply:
I could suggest `testCluster`. @nbalacha , any other suggestions?

Reply:
I see all the other tests are using clust so maybe we can continue with that.